misc/python/materialize/mzbuild.py

```diff
@@ -960,7 +992,7 @@ def publish_multiarch_images(
     for images in zip(*dependency_sets):
         names = set(image.image.name for image in images)
         assert len(names) == 1, "dependency sets did not contain identical images"
```

Comment:
```suggestion
        assert len(images) == 1, "dependency sets did not contain identical images"
```

Reply:
Do we actually want this change? Could there be duplicate images brought in as transitive dependencies? If so, the current way of creating a set of their names dedupes them.

Reply:
You're right, I was wrong!

Reply:
This is for publishing multiarch images, so it's expected that there are exactly two images with the same name, one for x86 and one for ARM. The code is structured such that it generalizes to more than two architectures, too!